course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -181,9 +181,64 @@ describe('Unit testing PeopleController', function() {
 
     });
     describe('confirmAddPeopleToCourse', function() {
-        it('reflects the right role and number of people');
-        it('initiates the process if confirmed');
-        it('doesn\'t initiate the process if canceled');
+        beforeEach(function() {
+            var ci = {
+                course_instance_id: $routeParams.courseInstanceId,
+                title: 'confirmAddPeopleToCourse test',
+            };
+            courseInstances.instances[ci.course_instance_id] = ci;
+            controller = $controller('PeopleController', {$scope: scope});
+        });
+
+        xit('reflects the right role and number of people', function() {
```

Comment:
I also have no idea how to do this, after some research. Let's just make it pending, and put some comments in it, maybe something like:
```
// todo: cannot yet find a way to access the modal's controller scope within a test
it('reflects the right role and number of people');
```